chips/ibex/src/i2c.rs

```diff
@@ -0,0 +1,8 @@
+use kernel::common::StaticRef;
+use lowrisc::i2c::{I2c, I2cRegisters};
+
+pub static mut I2C: I2c = I2c::new(I2C_BASE);
+
+// Random guess, this isn't documents
```

Comment:
Random guess that's working on FPGA image, or just a placeholder address?

Reply:
It's just a place holder. Although there is I2C RTL it doesn't seem to be exposed via MMIO in OT yet.

Reply:
Ah, so this fully isn't expected to work yet. Okay, for OT that's understandable. Can this comment be updated to reflect that this is a placeholder and there's not official MMIO interface (yet)?

Reply:
Done